bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieJournalTest.java

```diff
@@ -640,6 +637,15 @@ public void testTruncatedInEntryJournal() throws Exception {
         }
     }
 
+    private Bookie startBookieReadJournal(ServerConfiguration conf) throws IOException, InterruptedException, BookieException {
+        Bookie b = new Bookie(conf);
+        Journal journal = b.journals.get(0);
```

Comment:
wondering if this testsuite is configured to use multiple journal directories, will it work correctly?

Reply:
Updated the method to make it generic.